src/components/settings/PushNotifications/PushNotificationsBanner/index.tsx

```diff
@@ -86,6 +86,21 @@ export const _getSafesToRegister = (
   return { [chainId]: newlyAddedSafes }
 }
 
+const TrackBanner = (): null => {
+  const hasTracked = useRef(false)
+
+  useEffect(() => {
+    if (hasTracked.current) {
+      return
+    }
+
+    trackEvent(PUSH_NOTIFICATION_EVENTS.DISPLAY_BANNER)
```

Comment:
I suggest we rename the event action for this event to differentiate it from the events we've erroneously tracked so far.